### DISCUSSION TITLE
Set MinDays for specific date?

### DISCUSSION THREAD

**Discussion by whousehomestay** [easepick/easepick#38] [General]
Hello, is there a way to set min days for specific date (in hotel, we call it 'seasonal minimum nights')?
For example: we set default min days: 2, but in some dates ex. 20-04-2022 to 23-04-22 we set min days to 3.

I'm sorry for my bad english... ^_^

**Comment by wakirin** [comment]
Hello,
you can do something like this: https://jsfiddle.net/waki/a92xjk3g/

**Reply by whousehomestay** [reply]
Wow, very good, thank you....
